### PR TITLE
(fix: #350, #678) Change NodeJS version from "recommended" to "required"

### PR DESCRIPTION
**Pull request by haardikdharma10**
#### Summary 

This PR aims to solve #350, #678 

#### Ticket Link

https://github.com/mattermost/mattermost-developer-documentation/issues/350
https://github.com/mattermost/mattermost-developer-documentation/issues/678

### CODE REVIEW

**Comment by enahum** [comment]
We won't be accepting this PR, as there is an upcoming PR that should be ready next week that updates this and other parts of the mobile documentation.

cc: @hanzei

**Comment by hanzei** [comment]
@enahum I'm wondering if we would be open to accepting this PR as it's already an improvement of the existing docs. Not only does it make clear what NPM version is need, it also fixes a broken link and some typos. A future PR might also benefit from this.

**Comment by enahum** [comment]
@hanzei I rather have #709 merge first and then update this PR to add the changes that still apply

**Comment by hanzei** [comment]
That sound good to then. Then let's but this PR on the bench until https://github.com/mattermost/mattermost-developer-documentation/pull/709  merged.

**Comment by enahum** [comment]
mobile docs have been updated. this PR needs to resolved the conflicts

**Comment by jfrerich** [comment]
Hi @haardikdharma10! Do you need any assistance in resolving these conflicts?

**Comment by haardikdharma10** [comment]
Hi @jfrerich, yes that would be really helpful:)

**Comment by jfrerich** [comment]
I'll take a look at the conflicts and resolve them!  Thanks @haardikdharma10!

**Comment by haardikdharma10** [comment]
Thanks a lot @jfrerich!

**Comment by jfrerich** [comment]
Thanks @haardikdharma10.  I fixed the conflicts and added the appropriate viewers. They will take it from here!

File: site/content/contribute/mobile/developer-setup/_index.md [hunk 1]
```diff
@@ -17,15 +19,15 @@ A macOS computer is required to build the Mattermost iOS mobile app.
 
 Install the following prerequisite software to develop and build the iOS or Android apps. For macOS, we recommend using [Homebrew](https://brew.sh) as a package manager.
 
+#### Install [NodeJS](https://nodejs.org/en/).
+This includes NPM which is also needed. Currently, any version greater than or equal to 10.12.0 is **required** with npm 6.4.1 or higher. Please note that NodeJS version 11.x is **not** working.
+
```
Comment (enahum) [inline site/content/contribute/mobile/developer-setup/_index.md:24]: for me Node 12 & 14 are working just fine.

**Review by enahum — CHANGES_REQUESTED** [review]

File: site/content/contribute/mobile/developer-setup/_index.md (hunk 1 above)
Comment (jfrerich) [reply site/content/contribute/mobile/developer-setup/_index.md:24]: @haardikdharma10, after fixing the merge conflicts, I will leave you with the remaining feedback from @enahum!

Comment (haardikdharma10) [reply site/content/contribute/mobile/developer-setup/_index.md:24]: Sure thing @jfrerich, thanks a lot!

Comment (haardikdharma10) [reply site/content/contribute/mobile/developer-setup/_index.md:24]: Hi @enahum, what change do you expect here?

Comment (enahum) [reply site/content/contribute/mobile/developer-setup/_index.md:24]: @haardikdharma10 I would mention that node 12 and 14 are also working versions

Comment (haardikdharma10) [reply site/content/contribute/mobile/developer-setup/_index.md]: Thanks @enahum, Hope we can resolve this conversation now!

**Review by enahum — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Comment by hanzei** [comment]
Thanks for updating the docs @haardikdharma10 :+1:

**Pull request merged**